guardian/recipes-backend
# Fix eslint prettier integration

## What does this change?

When we merged https://github.com/guardian/recipes-backend/pull/65, we accidentally lost the necessary integration to run prettier as part of our eslint check! This PR fixes this by cherry picking two commits from that PR which re-enable the formatting check in CI.

## How to test

- check out the branch, add a commit with some weird formatting
- run `npm run lint` and verify that the check fails

I have also tested the CI check in https://github.com/guardian/recipes-backend/pull/73.

## How can we measure success?

- our new PRs won’t include formatting changes
  - because the strict check in CI will prevent us introducing non-standard formatting
- we won’t have to think about indentation/formatting when writing code
  - because we can enable prettier in our IDEs, which will automatically format it for us
